CATALOG.md

```diff
@@ -35,3 +35,71 @@ nativefier https://www.udemy.com/
 ### Notes
 
 Most videos will work, but to be sure everything works you are better off using the `--widevine` version AND signing the app afterwards. See this post: [#1147 (comment)](https://github.com/nativefier/nativefier/issues/1147#issuecomment-828750362)
+
+
+## Spotify
+
+```
+nativefier https://open.spotify.com/
+--name "Spotify" 
+--widevine 
+-u "<useragent your browser uses>"
+--inject spotify.js
+--inject spotify.css
+--icon icon.
+```
+
+### Notes
+
+- [Inject](https://github.com/nativefier/nativefier/blob/master/docs/api.md#inject) the folowing javascript as `spotify.js` to prevent "Unsupported Browser" messages.
```

Comment:
```suggestion
- [Inject](https://github.com/nativefier/nativefier/blob/master/docs/api.md#inject) the following javascript as `spotify.js` to prevent "Unsupported Browser" messages.
```